rpc/src/module/stats.rs

```diff
@@ -36,14 +40,22 @@ impl<CS: ChainStore + 'static> StatsRpc for StatsRpcImpl<CS> {
         let epoch = tip_header.epoch();
         let difficulty = tip_header.difficulty().clone();
         let is_initial_block_download = self.synchronizer.shared.is_initial_block_download();
+        let warnings = self
+            .alert_notifier
+            .lock()
+            .noticed_alerts()
+            .into_iter()
+            .map(|alert| alert.message.clone())
+            .collect::<Vec<String>>()
+            .join("\n");
 
         Ok(ChainInfo {
             chain,
             median_time: Timestamp(median_time),
             epoch: EpochNumber(epoch),
             difficulty,
             is_initial_block_download,
-            warnings: String::new(),
+            warnings,
```

Comment:
I think we can just use `alerts` here.

Reply:
Alert includes `signatures` and other fields which user does not care about, it's better to use another struct if we decide to expose more information.

But for now, I'm not sure what's field should be exposed other than a simple message.

Reply:
I' think we can return `id`, `received_at`, `notice_until` and `message` for now.

``` json
{
   id: 1,
   received_at: 1559641204,
   notice_until: 1559641204,
   message: "a alert message"
}
```

Reply:
I mean the field name